mods/other/rocket_launcher/init.lua

```diff
@@ -121,18 +117,61 @@ rocket.on_step = function(self, dtime, moveresult)
 			end
 		end
 	end
-
 	if moveresult.collides then
 		if can_explode(pos, self.puncher_name, self.radius)  then
 			tnt.boom(pos, {
 				radius = self.radius,
-				-- ignore_indestructible = true,
 				puncher_name = self.puncher_name
 			})
 		end
 		self.object:remove()
 	end
 end
 
-
 minetest.register_entity("rocket_launcher:rocket", rocket)
+
+ctf_modebase = ctf_modebase or {}
+ctf_modebase.update_wear = {}
+ctf_modebase.update_wear.active_updates = {}
+
+function ctf_modebase.update_wear.start_update(pname, toolname, wear_step, reset_on_done)
```

Comment:
You shouldn't manipulate namespace and API of another mod from this mod. And you shouldn't redefine the ctf_modebase API in this mod. If a change is needed for the API, you should do it in the ctf_modebase itself.